busybee/service.py

```diff
@@ -149,9 +151,23 @@ def register_modules(self):
         print("REGISTERING MODULES")
         print("###############")
         modules = self._mod_descriptors
+
         for module in modules.values():
             module_id = module["id"]
             module_descriptor = module["desc"]
+            if "mod-consortia" in modules and "mod-authtoken" in module_id:
+                env_vars = module_descriptor["launchDescriptor"]["env"]
+                for env_var in env_vars:
+                    if env_var["name"] == "JAVA_OPTIONS":
+                        env_var["value"] += " -Dallow.cross.tenant.requests=true"
+
+            if "mod-consortia" in module_id:
+                env_vars = module_descriptor["launchDescriptor"]["env"]
+                system_user_password_env = {"name": "SYSTEM_USER_PASSWORD", "value": "consortia-system-user"}
+                system_user_username_env = {"name": "SYSTEM_USER_NAME", "value": "consortia-system-user"}
+                env_vars.append(system_user_password_env)
+                env_vars.append(system_user_username_env)
+
```

Comment:
contains main logic, Other places reformatted based on official python style. these can be reverted, if you want.

Reply:
we have more that 7 modules that use system users, so in order to not modify source code each time we can create a file where specify env varaibles for each module and read that file here. But again to not block current consortia karate stories we can do it later, as @okolawole-ebsco prefer

Reply:
module specifc env vars is something I had thought about. I like the approach you describe @SerhiiNosko

Reply:
@azizbekxm some of the refactoring don't make sense i think that may be because of macOS/Linux line endings. We can reformat all the code according to a specific linter config defined in this repo later.

Reply:
I got it, reverted.